### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by YanXingyv** [TaatiTeam/MotionAGFormer#31]
Hello, may I ask whether the dataset generated by h36m.py in readme is GT or CPN

**Comment by SoroushMehraban** [comment]
Hi @YanXingyv 
It's none of them. MotionBERT uses Stacked Hourglass and we use the processed version of them. Unlike CPN provided by VideoPose3D paper, it also contains confidence scores.

**Comment by YanXingyv** [comment]
Thank you, there are p1, p2 and p1 on GT in the experimental data of the paper, but I did not output p1 on GT when evaluating your pre-training model, please ask how to output p1 on GT

**Comment by SoroushMehraban** [comment]
On config files we have `use_proj_as_2d` that is set to False by default. When you set it to True, It goes [here](https://github.com/TaatiTeam/MotionAGFormer/blob/master/data/reader/motion_dataset.py#L325) and construct the GT 2D pose.

Note that in the code it just throws away the Z-axis which is not technically the right thing to do. But since we normalize the input 2D pose, the result would be the same as applying camera projection matrix.

**Comment by YanXingyv** [comment]
Ok, I see, thank you very much
